app/scripts/views/mixins/experiment-mixin.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'underscore',
+  'lib/experiment',
+  'lib/url'
+], function (_, ExperimentInterface) {
+  'use strict';
+
+  return {
+    initExperiments: function () {
+      var experiments = new ExperimentInterface({
+        window: this.window,
+        able: this._able,
+        metrics: this.metrics,
+        account: this._account,
+        user: this.user
+      });
+      experiments.chooseExperiments();
+      return experiments;
+    },
+
+    openGmailTab: function () {
```

Comment:
@shane-tomlinson so I guess this will go into its own `lib/` module or mixin or something. Thoughts?